### DISCUSSION TITLE
webstorm  doesn't see variables

### DISCUSSION THREAD

**Discussion by m1x-developer** [mantinedev/mantine#6309] [Q&A]
I deployed this template and there were errors inside the css modules, how to fix it?

<img width="883" alt="image" src="https://github.com/mantinedev/mantine/assets/83704092/1723a89a-cfc9-4883-a359-c2ae999f4370">

<img width="1314" alt="image" src="https://github.com/mantinedev/mantine/assets/83704092/4467b753-8ba2-4fc6-a0aa-993d96beb173">

webstorm  doesn't see variables

**Comment by rtivital** [comment]
You need to setup postcss extension for Webstorm or disable css validation entirely.

**Reply by dfaivre-pcs** [reply]
PostCss extension doesn't seem to resolve it for me. Anyone figure out how to get it to work?
